### ISSUE TITLE
Update tns-core-modules to @nativescript/core

### ISSUE THREAD

**Issue by danielorihuela** [jibon57/nativescript-mediafilepicker#113]
Good morning jibon57,

Thanks for the development of an amazing plugin including camera and gallery functionalities, we find it very useful. However, your plugin currently depends on tns-core-modules (which seems deprecated: https://www.npmjs.com/package/tns-core-modules). Nativescript team recommends using @nativescript/core instead.

We would like to make a request of updating the imports to the recommended package by the Nativescript team, not by us.

Thanks for your time.

**Comment by mreall** [comment]
Nativescript 7 no longer supports tns-core-modules, so this update is critical for anyone upgrading. @jibon57 is this fix on your radar? We appreciate the work you have put into it.

**Comment by danielorihuela** [comment]
Not really, we were able to solve it by setting some aliases in our webpack configuration. So if anyone has a similar problem he can do that. Thanks for all @mreall.

**Comment by csimpi** [comment]
@mreall  @danielorihuela would you guys please share the solution? It's so bad there's a solution but the community can't see it :)

**Comment by danielorihuela** [comment]
@csimpi you have to go in to the alias section of your webpack config and add a new alias for `tns-core-modules`. 
Example: 

```
alias: {
    "tns-core-modules": "@nativescript/core",
    "nativescript-angular": "@nativescript/angular"
}
```

https://webpack.js.org/configuration/resolve/#resolvealias

**Comment by csimpi** [comment]
@danielorihuela Thank you! It seems this has already been a part of my webpack and still having issues with NS7 :(

**Comment by mreall** [comment]
Considering how long this and other issues have been open with no feedback from @jibon57, I ultimately had to go with another plugin.

**Comment by csimpi** [comment]
@mreall Is there any reliable file-picker plugin for nativescript? I didn't find any suitable replace.

**Comment by csimpi** [comment]
@mreall BTW, I'm trying to update this to NS7 I'll let you know if you can test my forked version. Maybe I can do it within a reasonable time.

**Comment by jibon57** [comment]
Hello everyone, I'm no longer using NativeScript now. Will be happy to get some contributor.

**Comment by csimpi** [comment]
@jibon57 - Thank you for the update! We're working on the migration with the NS community, I'll update you guys with the fork's address soon. Maybe tomorrow.
Basically forking this and making these changes can be a temporary solution:
https://nativescript.org/blog/nativescript-7-for-plugin-authors/
